examples/case_studies/conditional_autoregressive_priors.myst.md

```diff
@@ -0,0 +1,350 @@
+    beta1 = pm.Normal("beta1", mu=0.0, tau=1.0e-5)
+    # variance parameter of the independent random effect
+    tau_ind = pm.Gamma("tau_ind", alpha=3.2761, beta=1.81)
+    # variance parameter of the spatially dependent random effects
+    tau_spat = pm.Gamma("tau_spat", alpha=1.0, beta=1.0)
+
+    # prior for alpha
+    alpha = pm.Beta("alpha", alpha=1, beta=1)
+
+    # area-specific model parameters
+    # independent random effect
+    theta = pm.Normal("theta", mu=0, tau=tau_ind, dims="area_idx")
+    # spatially dependent random effect
+    phi = pm.CAR("phi", mu=np.zeros(N), tau=tau_spat, alpha=alpha, W=adj_matrix, dims="area_idx")
+
+    # exponential of the linear predictor -> the mean of the likelihood
+    mu = pm.Deterministic("mu", pt.exp(logE + beta0 + beta1 * x + theta + phi), dims="area_idx")
+
+    # likelihood of the observed data
+    pm.Poisson("y_i", mu=mu, observed=y, dims="area_idx")
+
+    # sampling the model
+    car_idata = pm.sample(
+        2000, tune=3000, cores=8, step=[pm.NUTS(max_treedepth=20, target_accept=0.8)]
+    )
+```
+
+```{code-cell} ipython3
+car_stacked = az.extract(car_idata)
+```
+
+Comparing the regression parameters $\beta_0$ and $\beta_1$ between the three models that we have fit, we can see that accounting for the spatial dependence between observations has the ability to greatly impact the interpretation of the effect of covariates on the response variable.
+
+```{code-cell} ipython3
+beta_density = az.plot_density(
+    [independent_idata, fixed_spatial_idata, car_idata],
+    data_labels=["Independent", "Spatial with alpha fixed", "Spatial with alpha random"],
+    var_names=["beta0", "beta1"],
+    shade=0.1,
+)
+```
+
+As you can see from the divergences given above, we have a difficult time effectively sampling from the third model. This is not an issue with PyMC, and is a common issue among software packages for models of this form: the model structure is unidentifiable. Though this should not be taken as guaranteed to be accurate, we can plot the marginal posterior for $\alpha$, and see that it is very near $1$.
+
+```{code-cell} ipython3
+az.plot_density([car_idata], var_names=["alpha"], shade=0.1)
+```
+
+This motivates having the ability that is equivalent to setting $\alpha=1$. In the definition of the CAR prior, $\alpha \in(0, 1)$. If $\alpha \rightarrow 1$, we get an alternate prior called the *intrinsic conditional autoregressive (ICAR)* prior. The ICAR prior has some desirable properties compared to the CAR prior, and is more widely used in spatial models, specifically the BYM {cite:p}`besag1991bayesian`, Leroux {cite:p}`leroux2000estimation` and BYM2 {cite:p}`riebler2016intuitive` models. Currently, work is being done to include the ICAR prior within PyMC.
+
++++
+
+## Authors
+
+* Adapted from a previous PyMC example notebook, authored by Junpeng Lao {ref}`conditional_autoregressive_model` by Conor Hassan on July, 2022.
+* Re-executed by Daniel Saunders in May, 2023
```

Comment:
I'd probably say something like:

```
* Adapted from {ref}`another PyMC example notebook <conditional_autoregressive_model>` by Conor Hassan and Daniel Saunders ([pymc-examples#417](https://github.com/pymc-devs/pymc-examples/pull/417) and [pymc-examples#547](https://github.com/pymc-devs/pymc-examples/pull/547/)).
```

Authorship of the other notebook should be on the other notebook I think.